quixstreams/dataframe/windows/definitions.py

```diff
@@ -23,6 +28,7 @@ def __init__(
         dataframe: "StreamingDataFrame",
         name: Optional[str] = None,
         step_ms: Optional[int] = None,
+        on_late: Optional[WindowOnLateCallback] = None,
```

Comment:
Maybe lets set the default here instead of running `or` in the `__init__`?
```suggestion
        on_late: WindowOnLateCallback = default_on_late_callback,
```